app/controllers/tutor_comments_controller.rb

```diff
@@ -0,0 +1,52 @@
+class TutorCommentsController < ApplicationController
+  before_action :set_tutor_comment, only: [:edit, :update, :destroy]
+
+  def new
+    @tutor_comment = TutorComment.new
+    @tutor = Tutor.find(params[:tutor])
+  end
+
+  def create
+    @tutor_comment = TutorComment.new(tutor_comment_params)
+
+    if @tutor_comment.save
+      redirect_to tutor_path(tutor_comment_params[:tutor_id])
+    else
+      @tutor = Tutor.find(tutor_comment_params[:tutor_id])
```

Comment:
QS: this line finds the current tutor, and when we `render :new`, it will automatically jump to tutor's new, instead of tutor_comments's new? if i understand it right.